lib/Frontend/Types.cpp

```diff
@@ -27,41 +26,34 @@ struct TypeInfo {
 };
 
 static const TypeInfo TypeInfos[] = {
-#define TYPE(NAME, ID, TEMP_SUFFIX, FLAGS) \
-  { NAME, FLAGS, TEMP_SUFFIX },
-#include "swift/Driver/Types.def"
+#define TYPE(NAME, ID, TEMP_SUFFIX, FLAGS) {NAME, FLAGS, TEMP_SUFFIX},
+#include "swift/Frontend/Types.def"
```

Comment:
Another file of "please don't reformat things you aren't changing".

Reply:
But I *did* change this file: I removed the driver namespace, because it moved to the frontend.
I thought the rule was: "Don't reformat files you don't change, but it's OK to reformat a file you do change." As I tried to explain in the PR preamble: there was a dilemma. I could undo the git-clang-format changes to ease reviewing, but then the next person who changed the file and ran git-clang-format would just change it again. What am I missing here?

Reply:
My understanding was that git-clang-format is only supposed to edit the lines you changed, but when you move a file it thinks the whole thing has changed. That's why it won't be a problem for the next person. But (cf. previous comment) maybe I'm wrong about that.

I suppose you could trick it into not doing this behavior by moving the files first in a separate commit that breaks things, then making the changes, and then squashing the rest of it back.

Reply:
I've gone back and manually minimized the diffs.